packages/admin/src/pages/scheme/[schemeId]/manage-editors/index.page.tsx

```diff
@@ -0,0 +1,96 @@
+import InferProps from '../../../../types/InferProps';
+import { SummaryList } from 'gap-web-ui';
+import Meta from '../../../../components/layout/Meta';
+import CustomLink from '../../../../components/custom-link/CustomLink';
+import {
+  Action,
+  UnformattedEditorRow,
+  getServerSideProps,
+} from '../editors.getServerSideProps';
+import { Row } from 'gap-web-ui/dist/cjs/components/summary-list/SummaryList';
```

Comment:
I reckon this might break in sandbox - need to export this type from gap-web-ui and import properly

Reply:
oops, good spot